src/shared/components/flexbox/FlexBox.tsx

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
I've found VBox and HBox to be very useful for building UIs. If you can point me to a place using an alternative I'll try it out.